queue_services/auth-queue/pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[tool.poetry]
+name = "auth-queue"
+version = "0.1.0"
+description = ""
+authors = ["\"BC Registries and Online Services\""]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+CacheControl = "0.14.0"
+Flask = "1.1.2"
+Jinja2 = "3.0.3"
+MarkupSafe = "2.0.1"
+Werkzeug = "1.0.1"
+attrs = "23.2.0"
+blinker = "1.8.2"
+cachetools = "5.3.3"
+certifi = "2024.7.4"
+charset-normalizer = "3.3.2"
+click = "8.1.7"
+expiringdict = "1.2.2"
+google-api-core = "1.34.1"
+google-auth = "2.28.2"
+google-cloud-pubsub = "2.20.2"
+googleapis-common-protos = "1.63.0"
+grpc-google-iam-v1 = "0.13.0"
+grpcio-status = "1.48.2"
+grpcio = "1.64.0"
+idna = "3.7"
+importlib-resources = "5.13.0"
+itsdangerous = "2.0.1"
+jaeger-client = "4.8.0"
+jsonschema = "4.17.3"
+launchdarkly-server-sdk = "8.2.1"
+msgpack = "1.0.8"
+opentracing = "2.4.0"
+pkgutil_resolve_name = "1.3.10"
+proto-plus = "1.23.0"
+protobuf = "3.19.6"
+pyRFC3339 = "1.1"
+pyasn1 = "0.6.0"
+pyasn1_modules = "0.4.0"
+pycountry = "23.12.11"
+pyrsistent = "0.20.0"
+python-dateutil = "2.9.0.post0"
+python-dotenv = "1.0.1"
+pytz = "2024.1"
+requests = "2.32.2"
+rsa = "4.9"
+semver = "3.0.2"
+sentry-sdk = "2.3.1"
+six = "1.16.0"
+threadloop = "1.0.2"
+thrift = "0.20.0"
+tornado = "6.4.1"
+urllib3 = "2.2.2"
+zipp = "3.19.1"
+
+# VCS dependencies
+sbc-common-components = { git = "https://github.com/bcgov/sbc-common-components.git", rev = "master", subdirectory = "python" }
+auth-api = { git = "https://github.com/seeker25/sbc-auth.git", rev = "20087", subdirectory = "auth-api" }
```

Comment:
are you plan to update python later on?

Reply:
I tried at first, but it was getting more involved than I wanted it to be, e.g. need to modify CI set up (not sure if we want to switch to CI in bcregistry-sre, as this one was setting up a db for tests)